templates/settings.php

```diff
@@ -115,7 +159,19 @@
 						<?php \esc_html_e( 'Blocklist', 'activitypub' ); ?>
 					</th>
 					<td>
-						<p class="description"><?php \printf( \__( 'To block servers, add the host of the server to the "<a href="%s">Disallowed Comment Keys</a>" list.', 'activitypub' ), admin_url( 'options-discussion.php#disallowed_keys' ) ); ?></p>
+						<p class="description">
+							<?php
+							\printf(
+								// translators:
+								\esc_html__( 'To block servers, add the host of the server to the "%1$sDisallowed Comment Keys%2$s" list.', 'activitypub' ),
+								\sprintf(
+									'<a href="%s">',
+									esc_url_raw( admin_url( 'options-discussion.php#disallowed_keys' ) )
+								),
+								'</a>'
+							);
```

Comment:
```suggestion
							echo \wp_kses_post(
								// translators: %s is a URL.
								\esc_html__( 'To block servers, add the host of the server to the "<a href="%s">Disallowed Comment Keys</a>" list.', 'activitypub' ),
								esc_url_raw( admin_url( 'options-discussion.php#disallowed_keys' ) )
							);
```

Better to include the HTML tags.